src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherAction.java

```diff
@@ -0,0 +1,36 @@
+package edu.cornell.kfs.fp.document.web.struts;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.struts.action.ActionForm;
+import org.apache.struts.action.ActionForward;
+import org.apache.struts.action.ActionMapping;
+import org.kuali.kfs.fp.document.web.struts.DisbursementVoucherForm;
+import org.kuali.rice.core.api.config.property.ConfigurationService;
+import org.kuali.rice.core.api.util.RiceConstants;
+import org.kuali.rice.kew.api.KewApiConstants;
+import org.kuali.rice.kns.datadictionary.KNSDocumentEntry;
+import org.kuali.rice.kns.web.struts.form.KualiDocumentFormBase;
+import org.kuali.rice.krad.service.KRADServiceLocator;
+import org.kuali.rice.krad.util.KRADConstants;
+
+
+public class RecurringDisbursementVoucherAction extends CuDisbursementVoucherAction {
+	protected static Log LOG = LogFactory.getLog(RecurringDisbursementVoucherAction.class);
+	
+	@Override
+    public ActionForward refresh(ActionMapping mapping, ActionForm form, HttpServletRequest request, HttpServletResponse response) throws Exception {
```

Comment:
This method seems identical to the super method except the form; given that the RecurringDisbursementVoucher form extends the CuDisbursementVoucherForm do we really need to override this method?